src/cli.ts

```diff
@@ -52,6 +52,9 @@ interface CommandOptions {
   outputFile: string
   validate: boolean | undefined
   mcType: Enums.ComponentType
+  workspace: string[]
```

Comment:
please move the worksapce-related options up so they appear right below the `omit`